.github/workflows/pages.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           key: venv-${{ hashFiles('poetry.lock') }}
 
       - name: Install the project dependencies
-        run: poetry install
+        run: poetry install --no-root
```

Comment:
I would rather mark the whole project as "no-package" like [here](https://github.com/margo/specification/pull/62/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R23) with the `[tool.poetry]` option `package-mode = false`.